lighthouse-core/lib/i18n.js

```diff
@@ -118,7 +134,7 @@ const _icuMessageInstanceMap = new Map();
  * @return {{formattedString: string, icuMessage: string}}
  */
 function _formatIcuMessage(locale, icuMessageId, icuMessage, values) {
-  const localeMessages = LOCALES[locale] || {};
+  const localeMessages = LOCALES[locale];
```

Comment:
`locale` is always a `LH.Locale`, so standing behind our type checking here by having no fallback :)